app/views/account/admin/issues/edit.html.slim

```diff
@@ -2,6 +2,7 @@
   = stylesheet_link_tag '//cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.css'
   = javascript_include_tag '//cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.js'
   = javascript_include_tag 'leaflet-google.js'
+  = javascript_include_tag 'https://maps.googleapis.com/maps/api/js?key=AIzaSyD9_Kz2vq3-WLfATRY1KglTzLTDG8idF-k&libraries=places'
```

Comment:
We have environment variable ENV['GOOGLE_MAPS_API_KEY']

Use it instead of explicit placing api key